nautobot/docs/user-guide/administration/tools/nautobot-server.md

```diff
@@ -224,14 +224,7 @@ nautobot=> \q
 
 ### `dumpdata`
 
-+/- 1.3.0
-    - `extras.job` should now be included in the dump (removed `--exclude extras.job` from the example usage)
-    - `django_rq` should now be excluded from the dump (added `--exclude django_rq` to the example usage)
 
-+/- 1.5.23
-    - We do not recommend at this time using `--natural-primary` as this can result in inconsistent or incorrect data for data models that use GenericForeignKeys, such as `Cable`, `Note`, `ObjectChange`, and `Tag`.
-    - We also do not recommend at this time using `--natural-foreign` as it can potentially result in errors if any data models incorrectly implement their `natural_key()` and/or `get_by_natural_key()` API methods.
-    - `contenttypes` must not be excluded from the dump (it could be excluded previously due to the use of `--natural-foreign`).
```

Comment:
Change to a `!!! warning` instead of removing?

Reply:
Makes sense, adding a warning, though I think `at this time` is a bit awkward and could be removed if you agree.

Reply:
Agreed.